<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlDialog</class>
 <widget class="QDialog" name="ControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Center</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="solutionsGB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Applications</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="solutions"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="browseBtn">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="solutionSettingsBtn">
          <property name="text">
           <string>Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="deleteSolutionBtn">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="shortcutsGB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Shortcuts</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListView" name="shortcuts"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="shortcutBtnLayout">
        <item>
         <widget class="QPushButton" name="addBtn">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shortcutSettingsBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="deleteShortcutBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>solutions</tabstop>
  <tabstop>browseBtn</tabstop>
  <tabstop>solutionSettingsBtn</tabstop>
  <tabstop>deleteSolutionBtn</tabstop>
  <tabstop>shortcuts</tabstop>
  <tabstop>addBtn</tabstop>
  <tabstop>shortcutSettingsBtn</tabstop>
  <tabstop>deleteShortcutBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ControlDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ControlDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
